backend/python/app/services/implementations/story_service.py

```diff
@@ -87,6 +92,27 @@ def import_story(self, details, file):
             self.logger.error(str(error))
             raise error
 
+    def export_story_translation(self, id):
```

Comment:
lets wrap the body of this in a try-except statement to handle any errors that might pop up